setup.py

```diff
@@ -40,7 +40,7 @@
     'babel~=2.3',
     'dnspython~=1.0',
     'lxml~=3.6',
-    'multidict~=2.0',
+    'multidict<5,>=4.0',
```

Comment:
Could the same not be achieved with `~=4.0`?

Reply:
Can we get away with ``>=2.0,<5``? ``2.x`` is known to work at least, I presume you tested ``4.0``, so I *guess* that ``3.0`` will be fine too. This allows operating on older systems, too.